lxd/db/cluster/entities.go

```diff
@@ -733,3 +733,240 @@ func GetEntityURLs(ctx context.Context, tx *sql.Tx, projectName string, entityTy
+
+// certificateIDFromURL gets the ID of a certificate from its URL.
+var certificateIDFromURL = fmt.Sprintf(`SELECT ?, identities.id FROM identities WHERE '' = ? AND '' = ? AND identities.identifier = ? AND identities.auth_method = %d`, authMethodTLS)
+
+// instanceIDFromURL gets the ID of an instance from its URL.
+var instanceIDFromURL = `SELECT ?, instances.id FROM instances JOIN projects ON instances.project_id = projects.id WHERE projects.name = ? AND '' = ? AND instances.name = ?`
+
+// instanceBackupIDFromURL gets the ID of an instance backup from its URL.
+var instanceBackupIDFromURL = `SELECT ?, instances_backups.id FROM instances_backups JOIN instances ON instances_backups.instance_id = instances.id JOIN projects ON instances.project_id = projects.id WHERE projects.name = ? AND '' = ? AND instances.name = ? AND instances_backups.name = ?`
+
+// instanceSnapshotIDFromURL gets the ID of an instance snapshot from its URL.
+var instanceSnapshotIDFromURL = `SELECT ?, instances_snapshots.id FROM instances_snapshots JOIN instances ON instances_snapshots.instance_id = instances.id JOIN projects ON instances.project_id = projects.id WHERE projects.name = ? AND '' = ? AND instances.name = ? AND instances_snapshots.name = ?`
+
+// networkIDFromURL gets the ID of a network from its URL.
+var networkIDFromURL = `SELECT ?, networks.id FROM networks JOIN projects ON networks.project_id = projects.id WHERE projects.name = ? AND '' = ? AND networks.name = ?`
+
+// networkACLIDFromURL gets the ID of a network ACL from its URL.
+var networkACLIDFromURL = `SELECT ?, networks_acls.id FROM networks_acls JOIN projects ON networks_acls.project_id = projects.id WHERE projects.name = ? AND '' = ? AND networks_acls.name = ?`
+
+// nodeIDFromURL gets the ID of a node from its URL.
+var nodeIDFromURL = `SELECT ?, nodes.id FROM nodes WHERE '' = ? AND '' = ? AND nodes.name = ?`
+
+// operationIDFromURL gets the ID of an operation from its URL.
+var operationIDFromURL = `SELECT ?, operations.id FROM operations LEFT JOIN projects ON operations.project_id = projects.id WHERE coalesce(projects.name, '') = ? AND '' = ? AND operations.uuid = ?`
+
+// storagePoolIDFromURL gets the ID of a storage pool from its URL.
+var storagePoolIDFromURL = `SELECT ?, storage_pools.id FROM storage_pools WHERE '' = ? AND '' = ? AND storage_pools.name = ?`
+
+// storageVolumeIDFromURL gets the ID of a storage volume from its URL.
+var storageVolumeIDFromURL = fmt.Sprintf(`
+SELECT ?, storage_volumes.id FROM storage_volumes
+	JOIN projects ON storage_volumes.project_id = projects.id
+	JOIN storage_pools ON storage_volumes.storage_pool_id = storage_pools.id
+	LEFT JOIN nodes ON storage_volumes.node_id = nodes.id
+WHERE projects.name = ? AND replace(coalesce(nodes.name, ''), 'none', '') = ? AND storage_pools.name = ? AND CASE storage_volumes.type WHEN %d THEN '%s' WHEN %d THEN '%s' WHEN %d THEN '%s' WHEN %d THEN '%s' END = ? AND storage_volumes.name = ?
```

Comment:
Can you move each of these AND statements onto their own indented line for readability.